istioctl/cmd/analyze.go

```diff
@@ -313,6 +314,8 @@ func Analyze() *cobra.Command {
 		"The duration to wait before failing")
 	analysisCmd.PersistentFlags().BoolVarP(&recursive, "recursive", "R", false,
 		"Process directory arguments recursively. Useful when you want to analyze related manifests organized within the same directory.")
+	analysisCmd.PersistentFlags().BoolVar(&ignoreInvalid, "ignore-invalid", false,
+		"Ignores invalid input documents, for cases where analyze should run only on k8s compliant inputs.")
```

Comment:
```suggestion
		"Don't complain about non-YAML, non-Kubernetes files.")
```

Reply:
In this case, we are handling both yaml and non-yaml files.  Files with non-yaml extensions are already excluded in the gather function.  This section handles files with the yaml extension which cannot be parsed by the k8s go-client.  One example would be an empty yaml document (usually part of a multi-document file, with sections separated by `---`).

Reply:
@therealmitchconnors My suggestion didn't say non-.yaml, it said non-YAML.  Feel free to ignore.  It is a small point of style, that flag descriptions should say what the flag does rather than how it works (`Ignores`).

Reply:
I have updated the terminology to say what the flag does.